ingenialink/ethercat/servo.py

```diff
@@ -50,11 +63,13 @@ class EthercatServo(Servo):
 
     def __init__(
         self,
-        slave: CdefSlave,
+        slave: "CdefSlave",
```

Comment:
I've never seen double quotes for forward reference of type:
https://peps.python.org/pep-0484/#runtime-or-type-checking

Reply:
black changes it automatically to double quotes